openapi.yml

```diff
@@ -1216,7 +1216,7 @@ components:
     RequestDRO:
       description: A request to create a DRO.  This has the same general structure as a DRO but doesn't have externalIdentifier and doesn't require the access subschema. If no access subschema is provided, these values will be inherited from the AdminPolicy.
       type: object
-      additionalProperties: false
+      additionalProperties: true
```

Comment:
Would you mind adding a comment about that here?

Reply:
Yup. Done.